docs/sources/structure/_index.md

```diff
@@ -54,27 +54,132 @@ Depending on your product design and maturity, you may not need every topic:
 Some topics are optional and are usually found in specific contexts.
 For example, the _Create_ and _Monitor_ topics are used in Grafana OnCall, but aren't used in Grafana Tempo.
 
+### Give prominence to important topics
+
+When you don't have a long top-level table of contents, you can include them at the top level as individual topics or modifications to the standard topic list.
+
+For example, _Metrics-generator_ and _TraceQL_ are two of the most viewed topics in Tempo documentation.
+_Metrics-generator_ is a top-level topic and _TraceQL_ is in the topic _Query with TraceQL_, a modification of the standard _Query data_ topic.
+
+### Avoid extra hierarchy
+
+When you only have a single topic, don't nest it inside a standard topic just to use the standard topic list.
```

Comment:
When I was rearranging the Loki docs, there were a couple of places where I ended up with a single topic under a heading.  And I'm OK with that, because 1) it matches the structure that we agreed upon when the DB writers were coming up with our information architecture and 2) it means that when I get around to adding additional topics under those headings, I won't have to move files and update a whole bunch of broken links.

Reply:
^^ agreed, I have one Configure Asserts topic but there will be more.